### PR TITLE
add verify SSL option

### PR DESCRIPTION
**Pull request by servusoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ pipeline:
     image: cschlosser/drone-ftps
     hostname: example.com:21
     secrets: [ ftp_username, ftp_password ]
-    secure: true
+    secure: true | false # true = use FTP(S), false = FTP without SSL
+    verify: true | false # true = strong SSL verification, false = supress SSL versification error
```
Comment (cschlosser) [inline README.md]: typo: verification

Comment (cschlosser) [reply README.md]: Please also add which is the default value

**Review by cschlosser — CHANGES_REQUESTED** [review]
Thank you. Please fix the two changes and I will merge it

File: README.md (hunk 1 above)
Comment (servusoft) [reply README.md]: the default value for `verify` ist `true`, like for `secure:`. 
https://github.com/christophschlosser/drone-ftps/pull/4/commits/1e86957f100fc0f5edaf9785079cf4946c89e9d3#diff-41c2c2edf223528193cd5163683cad3dR18

do you mean add the info about default value in README?

Comment (cschlosser) [reply README.md]: Yes exactly. If we add this sort of documentation we should also make it clear what is the default value in the README :)

**Comment by servusoft** [comment]
done,. the typo is gone  and default values are documented..

**Comment by cschlosser** [comment]
Thank you for your work.
Could you please add your new option to the offficial drone plugin documentation (https://github.com/drone/drone-plugin-index)

**Comment by servusoft** [comment]
look please here:
https://github.com/servusoft/drone-plugin-index/blob/master/content/christophschlosser/drone-ftps/index.md
if the descritption is Ok, I make a PR.

**Comment by cschlosser** [comment]
"transfer will fails due the certificat validation.. "
transfer will fail due to the certificate validation.
"In this case the validation may be suppressed:"
In this case the validation may be skipped:
"The SSL encryption mey be disabled, if the FTP server not supports SSL."
The SSL encryption may be disabled, if the FTP server does not support SSL.

I found a few typos. These are my suggestions to correct them. Otherwise it looks good, feel free to make a PR.

Thanks for your work

**Comment by servusoft** [comment]
thaks, the PR is created, but I have no reaction:
https://github.com/drone/drone-plugin-index/pull/90

I think, its, better, you make the PR.

**Comment by cschlosser** [comment]
Give it some time. If there's no reaction til next week we'll see what we can do.

**Pull request merged**